lr1110/Makefile.app

```diff
@@ -0,0 +1,52 @@
+LR1110_DIR = $(TOCK_USERLAND_BASE_DIR)/lr1110/lr1110
+SEEED_DIR = $(LR1110_DIR)/seeed
+
+# include changed headers and parameters
+override CFLAGS += -I$(LR1110_DIR)/inc_changed -DREGION_US_915 -DRP2_103 -DTASK_EXTENDED_2 -DLR11XX -DLR11XX_TRANSCEIVER
```

Comment:
Same here. I think the region setting needs a better solution

Reply:
Ideally we should prompt users to input their region code, but we discussed to agree that this approach will be sufficient for the current stage.

Reply:
Who discussed?

This isn't sufficient as you don't actually allow people to change it. It's set here and in the code and completely undocumented

Reply:
We will think about that a bit more